blade-graphics/src/gles/command.rs

```diff
@@ -234,6 +235,7 @@ impl super::PassEncoder<'_, super::RenderPipeline> {
             topology: map_primitive_topology(pipeline.topology),
             limits: self.limits,
             vertex_attributes: &pipeline.inner.vertex_attribute_infos,
+            color_targets: &pipeline.inner.color_targets,
```

Comment:
instead of adding the field to `super::PipelineEncoder`, we should use it to issue the relevant commands right here